svd-rs/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+- add missing fields in `Device`, require `version`, `description`, `address_unit_bits` and `width`,
+  also `scheme_version` is required, but skipped during (de)serialize
```

Comment:
Typo.

Reply:
Fix if you can, please. I don't understand what you saying about.

Reply:
Sorry 😁

`schema_version`

Reply:
Really. Fixed and squashed